app/build.gradle

```diff
@@ -13,6 +13,7 @@ android {
         targetSdkVersion 29
         versionCode 12
         versionName "1.2.1"
+        applicationIdSuffix ".debug"
```

Comment:
This is a `defaultConfig`, which means that you've just added `.debug` suffix to both debug / release builds. This should be in `debug {}` section.